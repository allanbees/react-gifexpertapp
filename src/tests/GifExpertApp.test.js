import { shallow } from 'enzyme';
import GifExpertApp from '../GifExpertApp';

describe('Tests for GifExpertApp', () => {

    test('Must render the component properly', () => {
        
        const wrapper = shallow( <GifExpertApp /> );

        expect( wrapper ).toMatchSnapshot();

    });

    test('Must display list of categories', () => {

        const categories = ['Programming', 'Valorant'];
        const wrapper = shallow( <GifExpertApp defaultCategories = { categories }/> );
        expect( wrapper.find('GifGrid').length ).toBe( categories.length );
    });

})