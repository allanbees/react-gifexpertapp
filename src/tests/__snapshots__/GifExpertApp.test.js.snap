// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests for GifExpertApp Must render the component properly 1`] = `
<Fragment>
  <h2>
    GiftExpertApp
  </h2>
  <AddCategory
    setCategories={[Function]}
  />
  <hr />
  <ol />
</Fragment>
`;
